packages/firestore/src/lite-api/settings.ts

```diff
@@ -123,17 +123,23 @@ export class FirestoreSettingsImpl {
       }
     }
 
-    this.experimentalForceLongPolling = !!settings.experimentalForceLongPolling;
-    this.experimentalAutoDetectLongPolling =
-      !!settings.experimentalAutoDetectLongPolling;
-    this.useFetchStreams = !!settings.useFetchStreams;
-
     validateIsNotUsedTogether(
       'experimentalForceLongPolling',
       settings.experimentalForceLongPolling,
       'experimentalAutoDetectLongPolling',
       settings.experimentalAutoDetectLongPolling
     );
+
+    if (!settings.experimentalForceLongPolling) {
```

Comment:
nit: Negations hurt readability IMHO. I think flipping the conditional structure is slightly cleaner code.